docs/translating/polish.md

```diff
@@ -2,6 +2,10 @@
 
 Use semi-formal Polish for translation, some specifics:
 
+* In polish websites it's more common to use just "search" ("szukaj") than, "search for people".  
+
+* When want to say "we", you would use "my" but, if want to say something is "our" you would use "nasze".
```

Comment:
I'm not sure this is relevant -- whenever you use plural "my", you'd use "nasze". I couldn't think of a specific example that would be unclear.